tests/test_async.py

```diff
@@ -23,9 +23,42 @@ def _predict(self, item, **kwargs):
 
     library = ModelLibrary(models=[SomeAsyncModel, ComposedModel])
     m = library.get("composed_model")
+    assert isinstance(m.model_dependencies["async_model"], WrappedAsyncModel)
     assert m.predict({"hello": "world"}) == {"hello": "world"}
 
 
+@pytest.mark.asyncio
+async def test_compose_async_sync_async(event_loop):
+    class SomeAsyncModel(AsyncModel):
+        CONFIGURATIONS = {"async_model": {}}
+
+        async def _predict(self, item):
+            await asyncio.sleep(0.1)
```

Comment:
```suggestion
            await asyncio.sleep(0)
```

Let's not waste 100m for no good reason. `asyncio.sleep(0)` is not a no-op, it still gives control back to the event loop

Reply:
I did not know that thanks!